views/default/manage.pdt

```diff
@@ -1,69 +1,65 @@
 <?php
+$link_buttons = [['name' => $this->_('CWatch.manage.add_module_row', true), 'attributes' => ['href' => $this->base_uri . 'settings/company/modules/addrow/' . $module->id]]];
 
-$link_buttons = array(['name'=>$this->_('CWatch.add_module_row', true), 'attributes'=>array('href'=>$this->base_uri . 'settings/company/modules/addrow/' . $module->id)]);
-    $this->Widget->clear();
-    $this->Widget->setLinkButtons($link_buttons);
-    $this->Widget->create($this->_('CWatch.manage.boxtitle', true, $this->Html->_($module->name, true)), ['id' => 'manage_none']);
+$this->Widget->clear();
+$this->Widget->setLinkButtons($link_buttons);
+$this->Widget->create($this->_('CWatch.manage.boxtitle', true, $this->Html->_($module->name, true)), ['id' => 'manage_none']);
 ?>
 
 <div class='title_row first'>
-    <h3><?php $this->_('CWatch.manage.module_row_title'); ?></h3>
+    <h3><?php $this->_('CWatch.manage.title'); ?></h3>
 </div>
 
-    <?php
-	$num_rows = count($this->Html->ifSet($module->rows));
-	
-	if ($num_rows > 0) {
-	?>
+<?php
+$num_rows = count($this->Html->ifSet($module->rows));
```

Comment:
This should default to an empty array rather than an empty string or it will not work on php 7.2+